libs/common/src/cast.helper.ts

```diff
@@ -130,16 +131,32 @@ export class ImageBase64Validator implements ValidatorConstraintInterface {
   }
 }
 
-// export const IsNotUUID = (validationOptions?: ValidationOptions): PropertyDecorator => (object: object, propertyName: string) => {
-//   registerDecorator({
-//     name: 'isNotUUID',
-//     target: object.constructor,
-//     propertyName,
-//     options: validationOptions,
-//     validator: {
-//       validate(value) {
-//         return !isUUID(value);
-//       }
-//     }
-//   });
-// };
\ No newline at end of file
+
+export class AgentSpinupValidator {
+  private static validateField(value: string, errorMessage: string): void {
+      if (!value) {
+          throw new BadRequestException(errorMessage);
+      }
+  }
+
+  private static validateWalletName(walletName: string): void {
```

Comment:
There is no need to validate the wallet name, as the agent accepts alphanumeric characters